spec/features/student_dashboard_view_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+
+describe 'viewing the dashboard as a student' do
+  let!(:student) { create :user, staff: false }
+  before :each do
+    when_current_user_is student, integration: true
+  end
+  context 'student has submitted an application' do
+    context 'student got an interview' do
+      let!(:interview) { create :interview, user: student }
+      before :each do
+        visit student_dashboard_url
+      end
+      it 'displays the date, time, and location of any interviews' do
+        expect page.has_text? interview.information
```

Comment:
This goes for several expectations I've read; would you like me to flag all of them, or are you fine going through and doing a general sweep?